modulefiles/module_base.orion

```diff
@@ -12,47 +12,49 @@ module load contrib noaatools
 ## load programming environment
 ## this typically includes compiler, MPI and job scheduler
 ##
+# Core modules
 module load intel/2018
 module load impi/2018
 
-#For ocean post: 
 module load ncl/6.6.2
 module load nco/4.8.1
 
-##
-## NCEP libraries (temporary version to match the CCPP requirements)
-##
+module load slurm/19.05.3-2
+# Stack modules
+
+module use /apps/contrib/NCEP/libs/modulefiles/stack
+module load hpc/1.0.0
+module load hpc-intel/2018.4
+module load prod_util/develop
+#module load hpc-impi/2018.4
+module load bacio
+#module load prod_util
+module load crtm
+module load ip
+module load sp
+module load w3nco
+module load g2
+module load g2tmpl
+module load jasper
+module load png
+module load netcdf_parallel/4.7.4
+module load hdf5_parallel/1.10.6
```

Comment:
these are not from the stack.

Reply:
As of I know these are required modules of this workflow.

Reply:
Those are for atm post

Reply:
The stack builds them too. you can (should) use the ones from the stack. That is the point of this PR, isn't it?

Reply:
Let me test the stack version